src/command_line_parser.cc

```diff
@@ -927,11 +928,12 @@ CLParser::ParseCommandLine(int argc, char** argv)
       {"endpoint", required_argument, 0, long_option_idx_base + 61},
       {"request-count", required_argument, 0, long_option_idx_base + 62},
       {"warmup-request-count", required_argument, 0, long_option_idx_base + 63},
-      {"schedule", required_argument, 0, long_option_idx_base + 64},
+      {"fixed-schedule", no_argument, 0, long_option_idx_base + 64},
```

Comment:
`--schedule=<n>,<n>,<n>,...` is replaced by `--fixed-schedule` (no values).

It signals PA to look for `timestamp` key in input data JSON.